packages/fleet/lib/models/nodes.ts

```diff
@@ -54,6 +55,8 @@ interface DBNode {
     readonly is_tracing_enabled: boolean;
     readonly is_profiling_enabled: boolean;
     readonly idle_max_duration_ms: number | null;
+    readonly execution_timeout_secs: number | null;
+    readonly provisioned_concurrency: number | null;
```

Comment:
These are required for function configuration.

- `execution_timeout_secs` is how long a function is allowed to run before it is terminated by the runtime
- `provisioned_concurrency` is a Lambda specific feature that helps mitigate cold-starts. It's configurable as we would likely want different values for different function profiles.